Tests/UnitTests/Core/Utilities/GridTests.cpp

```diff
@@ -1323,6 +1323,49 @@ BOOST_AUTO_TEST_CASE(closestPoints) {
   // clang-format on
 }
 
+BOOST_AUTO_TEST_CASE(grid_type_conversion) {
+  using EAxis = EquidistantAxis;
+  using VAxis = VariableAxis;
+
+  // Type conversion test
+  using Grid2Double = Grid<double, EAxis, VAxis>;
+  using Grid2Int = Grid<int, EAxis, VAxis>;
+
+  EAxis a(0.0, 1.0, 10u);
+  VAxis b({0., 1.2, 2.3, 3.4, 4.5, 5.6});
+  Grid2Double g2(std::make_tuple(a, b));
+  Grid2Double g2Copy(g2.axesTuple());
```

Comment:
i believe this will copy the tuple twice

Reply:
obsolete after change above.